packages/plugin-vue/src/main.ts

```diff
@@ -108,7 +109,7 @@ export async function transformMain(
   if (hasScoped) {
     attachedProps.push([`__scopeId`, JSON.stringify(`data-v-${descriptor.id}`)])
   }
-  if (devServer && !isProduction) {
+  if (devToolsEnabled || (devServer && !isProduction)) {
     // expose filename during serve for devtools to pickup
     attachedProps.push([`__file`, JSON.stringify(filename)])
```

Comment:
Due to safety concerns, the full path should not be used in production. Please use `path.basename(filename)`.
The development bundle should still use the full path so that users can open the file directly from devtools.

Reply:
👌

Reply:
Hi, I've just started using `@vitejs/plugin-vue2` in my rollup project, and was surprised to see the full filepath to my modules included in the generated output (as `__component__.options.__file`), and have not found a way to disable this. Can you advise on how to remove these filepaths?